Dockerfile.release

```diff
@@ -1,5 +1,10 @@
 FROM alpine:3.18.0
-RUN  apk add --no-cache wget ca-certificates
+RUN  apk update && apk add \
+  ca-certificates \
+  curl \
+  jq \
+  wget  \
+  && rm /var/cache/apk/*
```

Comment:
Can we not pass `--no-cache`? Or maybe I don't understand the motivation to clear the cache here.

Reply:
You are right... this would only be helpful for multiple `apk add` statements, not for a single one.

Reply:
@simar7 Could you have a look?

Reply:
+1